client/src/components/History/Multiple/MultipleViewItem.vue

```diff
@@ -75,14 +75,17 @@ export default {
         };
     },
     computed: {
-        ...mapState(useHistoryStore, ["getHistoryById"]),
+        ...mapState(useHistoryStore, ["getHistoryById", "loadHistoryById"]),
         sameToCurrent() {
             return this.currentHistory.id === this.source.id;
         },
         getHistory() {
             return this.getHistoryById(this.source.id);
         },
     },
+    mounted() {
+        this.loadHistoryById(this.source.id);
```

Comment:
The side effect of this is that your setting the history as the current history on the server, I think that can lead to some odd bugs if you have another tab open. I would suggest finding out whether the source history is loaded at all in the store, and why it doesn't have the hid count.

Reply:
On polling you may for instance need to include the hid count attribute in the server response. This would definitely benefit from a selenium test.

Reply:
The [`loadHistoryById` method](https://github.com/itisAliRH/galaxy/blob/fix-history-counter-empty/client/src/stores/historyStore.ts#L178) gets the extended history details with size and stores it in the store. But it should not change the current history. 

> On polling you may for instance need to include the hid count attribute in the server response. This would definitely benefit from a selenium test.

I don't understand it, sorry. Can you explain in more detail?

Reply:
I think you mean that we can get all the history changes, size and content items, from `watchHistory` which I've proposed as a fix in #16004, right? @mvdbeek